request-invoice/requestinvoice.py

```diff
@@ -1,125 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Why not moving this into the `/archive` like I did for the old `commando` plugin ( 748d50a ) instead?

Reply:
Because it is rather annoyingly useless and pretty much requires a full rewrite anyway.